xclim/testing/tests/test_run_length.py

```diff
@@ -158,19 +165,23 @@ def test_simple(self, tas_series, coord, expected, use_dask, ufunc):
 
 
 class TestWindowedRunEvents:
-    def test_simple(self):
+    @pytest.mark.parametrize("lastday", [True, False])
+    def test_simple(self, lastday):
         a = xr.DataArray(np.zeros(50, bool), dims=("x",))
         a[4:7] = True
         a[34:45] = True
-        assert rl.windowed_run_events(a, 3, dim="x") == 2
+        assert rl.windowed_run_events(a, 3, dim="x", lastday=lastday) == 2
 
 
 class TestWindowedRunCount:
-    def test_simple(self):
-        a = xr.DataArray(np.zeros(50, bool), dims=("x",))
+    @pytest.mark.parametrize("lastday", [True, False])
+    def test_simple(self, lastday):
+        a = xr.DataArray(np.zeros(50, bool), dims=("time",))
```

Comment:
I changed the dimension name from `x` to `time` so that tests passed, but I think the change that is actually needed is in `rle`, so that the new changes in this PR work with dimensions other than "time". 

The problematic lines are: 
https://github.com/UCL/xclim/blob/rle_lastday/xclim/indices/run_length.py#L88
`da = da.reindex(time=da.time[::-1])` 

and 

https://github.com/UCL/xclim/blob/rle_lastday/xclim/indices/run_length.py#L137
`d = d.reindex(time=d.time[::-1])`

Any ideas would be welcome! I'm not an expert on xarray, so there's hopefully a way to workaround hardcoding `time=` :)

Reply:
@raquel-ucl I'm pretty sure you can avoid the hard-coded `time`  and use the input `dim` param via:
````
da = da.reindex({dim:da[dim][::-1]})